tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py

```diff
@@ -595,6 +637,39 @@ def forward(self, x):
     return net, ref_model, {}
 
 
+def create_pytorch_module_with_optional_inputs_case1(tmp_dir):
+    net = make_pt_model_with_optional_input()
+    example_input = {"x": torch.zeros((1,3,10,10)), "y": torch.ones((1,3,10,10))}
+    ref_model = make_ref_pt_model_with_optional_inputs([-1, -1, -1, -1])
+    return net, ref_model, {"example_input": example_input}
+
+
+def create_pytorch_module_with_optional_inputs_case2(tmp_dir):
+    net = make_pt_model_with_optional_input()
+    example_input = {"x": torch.zeros((1,3,10,10)), "z": torch.ones((1,3,10,10))}
+    ref_model = make_ref_pt_model_with_optional_inputs([-1, -1, -1, -1], z_exist=True)
+    return net, ref_model, {"example_input": example_input}
+
+
+def create_pytorch_module_with_optional_inputs_case3(tmp_dir):
+    net = make_pt_model_with_optional_input()
+    example_input = {"x": torch.zeros((1,3,10,10)), "z": torch.ones((1,3,10,10))}
+    ref_model = make_ref_pt_model_with_optional_inputs([3, 3, 3, 3], z_exist=True)
+    return net, ref_model, {"example_input": example_input, "input_shape": [[3, 3, 3, 3], [3, 3, 3, 3]]}
```

Comment:
Is it a normal case that shapes of examples and `input_shape` to be different? What does it mean and is it described clearly in `help`?